src/Queue/StompQueue.php

```diff
@@ -488,9 +502,10 @@ protected function subscribeToQueues(): void
                 continue;
             }
 
-            $this->client->subscribe($queue, null, 'auto', [
+            $this->client->subscribe($queue, null, 'client', [
```

Comment:
export this to configuration?

Reply:
It can be done, but his should be default behaviour. Auto is causing nothing but trouble....
But OK, will prepare the option for config